dart/lib/src/sentry_attachment/io_sentry_attachment.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:io';
+
+import 'sentry_attachment.dart';
+
+class IoSentryAttachment extends SentryAttachment {
```

Comment:
Dart doesn't allow static methods nor factories in extensions. So we need to subclass here :(